lib/xclarity_client/xclarity_base.rb

```diff
@@ -16,7 +16,11 @@ def initialize(conf, uri)
     def connection_builder(conf, uri)
       $lxca_log.info "XClarityClient::XClarityBase connection_builder", "Creating connection to #{conf.host + uri}" 
       #Building configuration
-      @conn = Faraday.new(url: conf.host + uri) do |faraday|
+      host = conf.host;
+      if !host.start_with?('https://') && !host.start_with?('http://')
+         host = 'https://' + host
```

Comment:
Ok, but is there a possibility to use the HTTPS procol over other port? If yes, it could be a problem